packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -525,6 +526,56 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
 /// of [Curves.fastOutSlowIn].
 /// {@animation 250 266 https://flutter.github.io/assets-for-api-docs/assets/widgets/animated_container.mp4}
 ///
+/// {@tool sample --template=stateful_widget_scaffold}
```

Comment:
Would be great to illustrate what this animation looks like with a diagram linked from https://github.com/flutter/assets-for-api-docs/.